docs/diagrams/DeleteSequenceDiagram.puml

```diff
@@ -16,17 +16,17 @@ end box
 [-> LogicManager : execute("delete 1")
 activate LogicManager
 
-LogicManager -> AddressBookParser : parseCommand("delete 1")
-activate AddressBookParser
+LogicManager -> UniNurseParser : parseCommand("delete 1")
```

Comment:
`UniNurseParser` should be `UninurseBookParser` instead
```suggestion
LogicManager -> UninurseBookParser : parseCommand("delete 1")
```

Reply:
Updated